openedx/core/djangoapps/content/course_structures/tasks.py

```diff
@@ -0,0 +1,79 @@
+import json
+import logging
+
+from celery.task import task
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+
+log = logging.getLogger(__name__)
```

Comment:
Updated. Regardless of the logger name, I believe all logs are written to a single file: `[cms|lms]/edx.log`. At least that's what's happening on my sandbox.